API.md

```diff
@@ -0,0 +1,141 @@
+```
+```json
+{
+    "apiVersion": "rhjmc.redhat.com/v1alpha1",
+    "kind": "FlightRecorder",
+    "metadata": {
+        "creationTimestamp": "2020-01-15T22:54:22Z",
+        "generation": 1,
+        "labels": {
+            "app": "containerjfr"
+        },
+        "name": "containerjfr",
+        "namespace": "default",
+        "ownerReferences": [
+            {
+                "apiVersion": "v1",
+                "blockOwnerDeletion": true,
+                "controller": true,
+                "kind": "Service",
+                "name": "containerjfr",
+                "uid": "f895e203-37e9-11ea-8866-52fdfc072182"
+            }
+        ],
+        "resourceVersion": "298903",
+        "selfLink": "/apis/rhjmc.redhat.com/v1alpha1/namespaces/default/flightrecorders/containerjfr",
+        "uid": "f896b6e1-37e9-11ea-8866-52fdfc072182"
+    },
+    "spec": {
+        "port": 9091,
+        "recordingRequests": []
+    },
+    "status": {
+        "recordings": [],
+        "target": {
+            "apiVersion": "v1",
+            "kind": "Service",
+            "name": "containerjfr",
+            "namespace": "default",
+            "resourceVersion": "298899",
+            "uid": "f895e203-37e9-11ea-8866-52fdfc072182"
+        }
+    }
+}
+```
+
+## Creating a new Flight Recording
+
+To start a new recording, you will need to add a new recording request to the `FlightRecorder` in the `spec.recordingRequests` array. You can do this nicely on the command line with `oc edit`. The recording request must include the following:
+
+1. `name`: a string uniquely identifying the recording within that service
+2. `eventOptions`: an array of string options passed to Container JFR. The `"ALL"` special string can be used to enable all available events.
+3. `duration`: length of the requested recording as a [duration string](https://golang.org/pkg/time/#ParseDuration).
+
+```shell
+oc edit -o json flightrecorders/containerjfr
+```
+```json
+    "spec": {
+        "port": 9091,
+        "recordingRequests": [
+                {
+                        "name": "my-recording",
+                        "eventOptions": [ "jdk.SocketRead:enabled=true", "jdk.SocketWrite:enabled=true" ],
+                        "duration": "30s"
+                }
+        ]
+    },
+    "status": {
+        "recordings": [],
+```
+
+Once the operator has processed the request, it will be removed from the `spec.recordingRequests` array and a corresponding entry will appear in the `status.recordings` array. A recording is considered `active` if it is currently running.
+
+```shell
+$ oc get -o json flightrecorders/containerjfr
+```
+```json
+    "spec": {
+        "port": 9091,
+        "recordingRequests": []
+    },
+    "status": {
+        "recordings": [
+            {
+                "active": true,
+                "duration": "30s",
+                "name": "my-recording",
+                "startTime": "2020-01-23T15:55:04Z"
+            }
+        ],
```

Comment:
Same here, looks like some missing braces

Reply:
Should also be fixed.